### DISCUSSION TITLE
Bitclout Galleries.

### DISCUSSION THREAD

**Discussion by jam2ja** [deso-protocol/dips#122] [Medium CIPs]
Create system so that once someone has joined bitclout, they are able to create a collaborative gallery with with other biclouters within bitclout. To start off with it would be best to keep splits equal, yet over time this tool could be more sophisticated. 

This would be good tool for NFT's, gallery can publish with the artist, in an open and transparent way. When the NFT sells from the gallery all parties are paid at point of sale.

Often artists have to wait to paid till the financial situation is good for the gallery/publisher. 

Why? This would allow people to make works together. This is essentially something similar to what organisations like ASCAP and BMI are already doing. 
What do they do? For allowing people to use their content and get paid under the laws of copyright, they are taking a percentage for doing that work, and then passing what is left over onto the artist. Mostly these rights organisations are non-profit organisations. They take a cost for verification and admin of creative works.  However as most works publicly broadcast are owned by fewer companies, they have less relevance to self publishing artists.

**Comment by tijno** [comment]
Cool idea and for sure needed.

But personally I feel this is better done as a tool outside of bitclout rather then inside bitclout.com.

I think it would require some changes to the protocol maybe - for example how to ensure a NFT is sold through the galary ? Maybe if they were the owner of the NFT this could be managed that way.

**Comment by jam2ja** [comment]
Maybe within the node the artist could have the option to mint with a gallery/company/dao of choice.

Then back to the beginning on this conversation in previous thread.. where when people would sign up they could choose the possibility of creating a "gallery/company/dao" or "personal profile".





Bitclout Galleries.

When profiles are created. Another option would exist. Create a Gallery.
Any user has the possibility of creating a gallery.

After the process of minting NFT's is complete.

An option would exist where the creator of the NFT is able to send the NFT to any of these galleries.

The gallery gets a notification that work has been sent for display.
The gallery has to option to accept or reject the work. Or the gallery might accept all work.

Best make it clear to the creator of the NFT that when they send/share this work to the gallery, it is for public display and they have control of all appropriate rights to display or broadcast the created content publicly. This would allow these display galleries to project the content into many metaverses.

**Comment by jam2ja** [comment]
> for example how to ensure a NFT is sold through the galary ?

I had this idea that the work would remain in gallery for 30,60,90 days. "if the system i just described was in place" If it did not sell, it would bounce back to the creators wallet. yet that is quite a complexed smart contract?? thing to do at this point... or not?  Could have it so you could enter, primary, secondary... and so on minters.

> But personally I feel this is better done as a tool outside of bitclout rather then inside bitclout.com.

It would work better for business people if it was that way, yet for self publishers and artist collaborations the way i describe gives control to the artist, not the publisher node. Development so far with this platform feels top down, not bottom up. Go with bottom up, and the artists will see the merits of the site. not elon musk..
